components/03-modules/header/_search-form.scss

```diff
@@ -55,8 +56,8 @@ $search-form__button-icon-color    : $gray-dark !default;
         top: 0;
         right: 0;
         height: $search-form__input-height;
-        margin-bottom: 0;
-        border: none;
+        margin: $search-form__button-margin;
+        border: $search-form__button-border;
         border-radius: $search-form__height;
         background-color: $search-form__button-background;
         &:focus,
```

Comment:
shouldn't it exist as a separate module? what do you think?

Reply:
I talked with igloczek before, this way is ok.

Reply:
👍